main.go

```diff
@@ -216,17 +207,17 @@ OPTIONS:
 				},
 			},
 			plugin.Command{
-				Name:     "usb delete-driver-endpoint",
+				Name:     "usb-delete-driver-endpoint",
 				HelpText: "Delete a driver instance",
 				UsageDetails: plugin.Usage{
-					Usage: "cf usb delete-driver-endpoint NAME",
+					Usage: "cf usb-delete-driver-endpoint NAME",
 				},
 			},
 			plugin.Command{
-				Name:     "usb update-driver-endpoint",
+				Name:     "usb-update-driver-endpoint",
 				HelpText: "Update a driver instance",
 				UsageDetails: plugin.Usage{
-					Usage: `cf usb update-driver-endpoint NAME [-t ENDPOINT_URL] [-k AUTHENTICATION_KEY] [-c METADATA_AS_JSON]
+					Usage: `cf usb-update-driver-endpoint NAME [-t ENDPOINT_URL] [-k AUTHENTICATION_KEY] [-c METADATA_AS_JSON]
```

Comment:
The help text below still says `cf usb update-driver-endpoint…`; see also the other help texts.

Reply:
Will fix.

Reply:
Done with 563bb1b72c3fa3ec8f806e6e4937f56b528ab47a